404 notifier (#1381)
redpanda-data/documentation: modules/deploy/pages/redpanda/manual/disaster-recovery/index.adoc

= Disaster Recovery

> :description: Set up disaster recovery for Redpanda clusters using Shadowing for cross-region replication.

Reviewer: ```suggestion
:description: Learn about Shadowing with cross-region replication for disaster recovery.
```
